### PR TITLE
feat(notifications): remove tinygesture

### PR DESCRIPTION
**Pull request by Lakate**

### CODE REVIEW

**Comment by Lakate** [comment]
I added a horizontal Swiper with the necessary code and removed the import of the tinygesture library

File: src/components/Notification/components/HorizontalSwiper/HorizontalSwiper.ts [hunk 1]
```diff
@@ -0,0 +1,195 @@
+export interface Options {
```
Comment (Ruminat) [inline src/components/Notification/components/HorizontalSwiper/HorizontalSwiper.ts:1]: I think we should mention the source with this url — https://github.com/sciactive/tinygesture/blob/master/src/TinyGesture.ts

**Comment by ogonkov** [comment]
Do not add space symbol after `feat` and before opening parenthesis

**Comment by Ruminat** [comment]
The `tinygesture` library is still in package.json and package-lock.json

**Review by Ruminat — APPROVED** [review]

**Review by korvin89 — APPROVED** [review]

**Pull request merged**